### PR TITLE
@Omo-Tines Add paddle.Tensor Instance Methods to Paddle Frontend #15115

### PR DESCRIPTION
**Pull request by Omo-Tines**
added paddle_median instance

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by jkeane508** [comment]
Can you please link what issue this tackles?

Thanks!

**Comment by Omo-Tines** [comment]
Median

**Comment by Omo-Tines** [comment]
Any idea where i can tackle these errors from

**Comment by Omo-Tines** [comment]
#16639

File: ivy/functional/frontends/paddle/tensor/stat.py [hunk 1]
```diff
@@ -6,15 +6,23 @@
 )
 
 
+
+@with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
```
Comment (jkeane508) [inline ivy/functional/frontends/paddle/tensor/stat.py:10]: Not sure why you've added this in here

File: ivy/functional/frontends/paddle/tensor/stat.py [hunk 2]
```diff
@@ -6,15 +6,23 @@
...
+
 @to_ivy_arrays_and_back
 def mean(input, axis=None, keepdim=False, out=None):
     ret = ivy.mean(input, axis=axis, keepdims=keepdim, out=out)
     ret = ivy.expand_dims(ret, axis=-1) if ret.ndim == 0 else ret
     return ret
 
 
+
+@with_unsupported_dtypes({"2.4.2 and below": ("complex", "int8")}, "paddle")
```
Comment (jkeane508) [inline ivy/functional/frontends/paddle/tensor/stat.py:22]: Same with this

File: ivy/functional/frontends/paddle/tensor/stat.py [hunk 3]
```diff
@@ -25,9 +33,18 @@ def numel(x, name=None):
     return ivy.array([prod if prod > 0 else ivy.array(length, dtype=ivy.int64)])
 
 
+@with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
+@to_ivy_arrays_and_back
+def median(y, axis=None, keepdim=False,out=None):
+    mid = ivy.median(y, axis=axis, keepdims=keepdim,out=out)
+    #mid = ivy.expand_dims(mid, axis=-1) if mid.ndim == 0 else mid
```
Comment (jkeane508) [inline ivy/functional/frontends/paddle/tensor/stat.py:40]: We can remove comments

File: ivy_tests/test_ivy/test_functional/test_core/test_statistical.py [hunk 4]
```diff
@@ -166,7 +166,41 @@ def test_mean(
         axis=axis,
         keepdims=keep_dims,
     )
+# median
```
Comment (jkeane508) [inline ivy_tests/test_ivy/test_functional/test_core/test_statistical.py:169]: I dont think the frontend tasks require you to add tests for Ivy's API aswell

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Hey, thanks for opening the PR!

Some changes needed before we can get this merged :)

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

File: ivy/functional/frontends/paddle/tensor/stat.py (hunk 3 above)
Comment (Omo-Tines) [reply ivy/functional/frontends/paddle/tensor/stat.py:40]: Alright will do thank you

File: ivy/functional/frontends/paddle/tensor/stat.py (hunk 2 above)
Comment (Omo-Tines) [reply ivy/functional/frontends/paddle/tensor/stat.py:22]: so i remove all code in this snip?

File: ivy/functional/frontends/paddle/tensor/stat.py (hunk 1 above)
Comment (Omo-Tines) [reply ivy/functional/frontends/paddle/tensor/stat.py:10]: will remove shortly then

File: ivy_tests/test_ivy/test_functional/test_core/test_statistical.py (hunk 4 above)
Comment (Omo-Tines) [reply ivy_tests/test_ivy/test_functional/test_core/test_statistical.py:169]: oh i did not know that so I should remove the median tst?

**Comment by Omo-Tines** [comment]
Please Review @jkeane508

**Comment by jkeane508** [comment]
ivy-gardener

**Review by jkeane508 — CHANGES_REQUESTED** [review]
test_dot seems to be failing [here](https://github.com/unifyai/ivy/actions/runs/5463170502/jobs/9943519355?pr=16978#step:3:1292)

File: ivy_tests/test_ivy/test_functional/test_core/test_statistical.py [hunk 5]
```diff
@@ -168,6 +168,43 @@ def test_mean(
     )
 
 
+# median
```
Comment (jkeane508) [inline ivy_tests/test_ivy/test_functional/test_core/test_statistical.py:171]: You can also just remove this from the PR

**Review by jkeane508 — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_functional/test_core/test_statistical.py (hunk 5 above)
Comment (Omo-Tines) [reply ivy_tests/test_ivy/test_functional/test_core/test_statistical.py]: It has been removed

**Comment by Omo-Tines** [comment]
Please review @jkeane508

**Comment by Omo-Tines** [comment]
Im sorry but i dont understand the request. Is there a problem within the file that I have attended too?

**Comment by jkeane508** [comment]
> Im sorry but i dont understand the request. Is there a problem within the file that I have attended too?
Hey,

There seems to be some merge conflicts on your branch. Could you please resolve them so we can continue reviewing :)

Thanks!

**Comment by Omo-Tines** [comment]
It’s not allowing me to resolve the conflicts anymore

**Pull request closed without merging**